schemas/org.gnome.shell.extensions.blur-my-shell.gschema.xml

```diff
@@ -269,6 +269,11 @@
             <default>false</default>
             <summary>Boolean, whether to disable blur from this component when opening the overview or not</summary>
         </key>
+        <!-- CORNER RADIUS -->
+        <key type="i" name="radius">
```

Comment:
FWIW, Gnome 46 will replace `sigma` with `radius` in blur APIs. I'm not sure if we are going to reserve the `radius` key for Gnome 46+, or just keep the sigma key untouched, and assume `radius =  sigma * 2`. See https://github.com/aunetx/blur-my-shell/issues/532#issuecomment-1962274550

Reply:
I think, we could use `radius` for blur settings and `corner_radius` for the rounding effect in GNOME 46.

Reply:
Yes! I think we should use `corner_radius` and replace `sigma` by `radius` starting from GNOME 45, so that no change is required for it afterwards. Once this PR is merged, I will add the changes for GNOME 46 for the other blur effect!